intro-to-sql/sql-fixtures/create-products-table.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE products (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(32),
+  description TEXT,
+  publisher_id INT
+);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 1', 'some description 1', 1);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 2', 'some description 2', 1);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 3', 'some description 3', 2);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 4', 'some description 4', 3);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 5', 'some description 5', 3);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 6', 'some description 6', 103);
+
+INSERT INTO products (
+  name,
+  description,
+  publisher_id
+)
+VALUES ('product 7', 'some description 7', 105);
+
```

Comment:
There's a question asking for products with 1 or 2 digit publisher ids, but no 2 digit publisher ids. Adding this _should_ be ok, since the test result is read directly from the data.
```suggestion

INSERT INTO products (
  name,
  description,
  publisher_id
)
VALUES ('product 8', 'some description 8', 99);

```